new-log-viewer/src/index.css

```
@@ -3,3 +3,17 @@ html, body, #root {
     height: 100%;
     width: 100%;
 }
+
+html {
+    font-size: 16px;
+}
+
+:root {
+    /* font-family globals */
+    --ylv-ui-font-family: -apple-system, BlinkMacSystemFont, system-ui, Ubuntu, "Droid Sans", Roboto;
+
+    /* z-index globals */
+    /* .monaco-editor .minimap { z-index: 5; } */
```

```suggestion
    /* z-index globals
     *
     * Other z-index values in the project for reference:
     * ```
     * .monaco-editor .minimap { z-index: 5; }
     * ```
     */
```

The code block is a bit extra, but I'm trying to format the comment in a way that a markdown formatter / linter won't reformat it.

---

Gotcha